README.md

```diff
@@ -1,125 +1,112 @@
+<!--! [TOC] -->
 
-| Library | Description |
-| -- | -- |
-| **Cesium3DTiles** | Lightweight 3D Tiles classes. |
-| **Cesium3DTilesReader** | 3D Tiles deserialization, including 3D Tiles extension support. |
-| **Cesium3DTilesWriter** | 3D Tiles serialization, including 3D Tiles extension support. |
-| **Cesium3DTilesSelection** | Runtime streaming, decoding, level of detail selection, culling, cache management, and decoding of 3D Tiles. |
-| **CesiumAsync** | Classes for multi-threaded asynchronous tasks. |
-| **CesiumGeometry** | Common 3D geometry classes; and bounds testing, intersection testing, and spatial indexing algorithms. |
-| **CesiumGeospatial** | 3D geospatial math types and functions for ellipsoids, transforms, projections. |
-| **CesiumGltf** | Lightweight glTF processing and optimization functions. |
-| **CesiumGltfReader** | glTF deserialization / decoding, including glTF extension support (`KHR_draco_mesh_compression` etc). |
-| **CesiumGltfWriter** | glTF serialization / encoding, including glTF extension support. |
-| **CesiumIonClient** | Functions to access [Cesium ion](https://cesium.com/cesium-ion/) accounts and 3D tilesets using ion's REST API. |
-| **CesiumJsonReader** | Reads JSON from a buffer into statically-typed classes. |
-| **CesiumJsonWriter** | Writes JSON from statically-typed classes into a buffer. |
-| **CesiumUtility** | Utility functions for JSON parsing, URI processing, etc. |
+## 📖About
 
+Cesium Native is a set of C++ libraries for 3D geospatial, including:
 
-### 📗License
+* [3D Tiles](https://github.com/CesiumGS/3d-tiles) runtime streaming
+* lightweight glTF serialization and deserialization
+* high-precision 3D geospatial math types and functions, including support for global-scale WGS84 ellipsoids.
+* support for draping raster overlays from WMS, TMS, WMTS, and other sources over 3D tilesets
+
+Cesium Native powers Cesium's runtime integrations for [Cesium for Unreal](https://github.com/CesiumGS/cesium-unreal), [Cesium for Unity](https://github.com/CesiumGS/cesium-unity), [Cesium for Omniverse](https://github.com/CesiumGS/cesium-omniverse), and [Cesium for O3DE](https://github.com/CesiumGS/cesium-o3de). Cesium Native is the foundational layer for any 3D geospatial software, especially those that want to stream 3D Tiles. See [Projects Using Cesium Native](#projects-using-cesium-native) for a list of projects currently integrating with Cesium Native.
+
+![Cesium Platform and Ecosystem](./doc/img/integration-ecosystem-diagram.png)
+
+<p align="center"><em>A high-level Cesium platform architecture with the runtime integrations powered by Cesium Native and streaming content from Cesium ion.</em></p>
+
+## 📷Screenshots
+
+<table>
+  <tbody>
+    <tr>
+      <td>
+        <!--! \if DOXYGEN_EXCLUDE -->
+        <img src="doc/img/screenshot-googleplex.jpg" alt="Googleplex" />
+        <!--! \endif -->
+        <!--! \image html screenshot-googleplex.jpg -->
```

Comment:
You should be able to just use a markdown image rather than these conditionals. I tried it out in Doxygen and it seems to work. I haven't tried on GitHub but I can't think of any reason it wouldn't work there, too.
```suggestion
        ![Googleplex](doc/img/screenshot-googleplex.jpg)
```

Reply:
Unfortunately GitHub is what breaks here. For whatever reason, without some weird spacing tweaks, GitHub will just render the Markdown image tag here as plaintext. And with some weird spacing tweaks, [as Janine noted here](https://github.com/CesiumGS/cesium-native/pull/1024#discussion_r1871540966), it causes noticeable whitespace to show up in the output. But we can't just use the `<img>` tag, because Doxygen won't remap the URL if we do that.